test/just.test.js

```diff
@@ -0,0 +1,30 @@
+const just = require("../src/just")
+const qunit = require('qunit')
+
+const inc = x => x + 1
```

Comment:
style-wise, since the code base uses semicolons, I think the tests should follow suit (sorry!)

Reply:
I see that the project has an `.editorconfig`, but it only helps with a few things. Maybe we could use some `prettier` config file